### PR TITLE
manual for operators

### PR DESCRIPTION
**Pull request by XuHuaiyu**

### CODE REVIEW

File: sql/README.md [hunk 1]
```diff
@@ -10,62 +10,62 @@ TiDB 支持 SQL92 标准并兼容 MySQL 语法，为了帮您更好地使用 TiD
...
-	- Cast函数和操作符
-	- 位函数和操作符
-	- 加密和压缩函数
-	- 信息函数
-	- JSON 函数
-	- 全局事务 ID 函数
-	- GROUP BY 聚合函数
-	- 其他函数
-	- 精度数学
+  - [函数和操作符概述](sql/functions-and-operators/operators.md)
```
Comment (queenypingcap) [inline sql/README.md]: 操作符 or 运算符？Please be consistent

File: sql/functions-and-operators/operators.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+| [`\|\|`, `OR`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_or) | 逻辑或                                      |
+| [`+`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_plus) | 加                                        |
+| [`REGEXP`](https://dev.mysql.com/doc/refman/5.7/en/regexp.html#operator_regexp) | 使用正则表达式进行模式匹配                            |
+| [`>>`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_right-shift) | 右移                                       |
+| [`RLIKE`](https://dev.mysql.com/doc/refman/5.7/en/regexp.html#operator_regexp) | REGEXP 同义词                               |
+| [`*`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_times) | 乘                                        |
+| [`-`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_unary-minus) | 取反符号                                     |
+| [`XOR`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_xor) | 逻辑亦或                                     |
```
Comment (andelf) [inline sql/functions-and-operators/operators.md]: typo

Comment (XuHuaiyu) [reply sql/functions-and-operators/operators.md]: I can't get your point?

**Comment by XuHuaiyu** [comment]
dup with #307

**Pull request closed without merging**